packages/openapi-generator/src/generator.ts

```diff
@@ -101,6 +102,26 @@ async function generateSDKSources(
   }
 }
 
+async function createApis(
+  serviceDir: string,
+  openApiDocument: OpenApiDocument
+) {
+  return openApiDocument.tags.map(tag => {
+    const matchedOperations = openApiDocument.operations.filter(o =>
+      o.tags?.includes(tag)
+    );
+    const fileName = `${voca.kebabCase(tag + 'Api')}.ts`;
+
+    createFile(
```

Comment:
This could be dangerous in case I await the function I get the list of files, but the actial creation is not awaited. Also add the return type to the method signature.

Reply:
I'll refactor it